addons/pinniped/post-deploy/main.go

```diff
@@ -24,6 +24,9 @@ func main() {
 	// optional
 	flag.StringVar(&vars.PinnipedAPIGroupSuffix, "pinniped-api-group-suffix", vars.PinnipedAPIGroupSuffix, "The API group suffix used to talk to Pinniped APIs")
 
+	// optional
+	flag.BoolVar(&vars.PinnipedConciergeIsClusterScoped, "pinniped-concierge-is-cluster-scoped", vars.PinnipedConciergeIsClusterScoped, "Whether the Pinniped Concierge APIs are cluster-scoped")
```

Comment:
ok thanks for the convo - i updated the flag (and the pr description). wanna take a look and lmk if there is anything else you wanna change?

Reply:
oh interesting...yeah that one is interesting because on the client side we are just referring to the concierge's API group suffix, because we don't call supervisor APIs. but in the post-deploy job, we are referring to both the supervisor's _and_ concierge's API group suffix...meh naming is hard

Reply:
aaah gotcha, thanks for talking through that.  it probably makes sense to leave those as is with that rationale.